### PR TITLE
Fix check_service returning Warning instead of Critical

### PR DESCRIPTION
**Pull request by Crunsher**
Still need to test this before merging.

refs #5849

### CODE REVIEW

File: plugins/check_service.cpp [hunk 1]
```diff
@@ -126,9 +126,7 @@ INT parseArguments(INT ac, WCHAR **av, po::variables_map& vm, printInfoStruct& p
 
 	printInfo.service = vm["service"].as<std::wstring>();
 
-	if (vm.count("warn")) {
-		printInfo.warn = true;
-	}
+	vm.count("warn") ? printInfo.warn = true : printInfo.warn = false;
```
Comment (Crunsher) [inline plugins/check_service.cpp]: printInfo = vm.count("warn")?

**Comment by mcktr** [comment]
I tested this PR:

```
.\check_service.exe -s w32time
SERVICE "w32time"CRITICAL NOT RUNNING | service=1;;;1;7
.\check_service.exe -s w32time -w
SERVICE "w32time"WARNING NOT RUNNING | service=1;;;1;7
```
And the output seems to be correct now.

File: plugins/check_service.cpp (hunk 1 above)
Comment (mcktr) [reply plugins/check_service.cpp]: I'll guess you mean `printInfo.warn = vm.count("warn");`? ;)

I also tested this variant and it worked as expected. I would stick to this variant, as it "feels" somehow cleaner.

**Comment by Crunsher** [comment]
Tested it again just now to make sure it works 👌

**Comment by mcktr** [comment]
@Crunsher This PR has no milestone yet, I propose that this should be in v2.8.1? :)

**Comment by gunnarbeutner** [comment]
Good catch. Moved to 2.9.0 for now - until we've decided what to backport into 2.8.1. :)

**Pull request merged**